core/services/blockhashstore/feeder_test.go

```diff
@@ -236,6 +241,128 @@ var (
 	}
 )
 
+// Context returns a context with the test's deadline, if available.
+func GetTestContext(t *testing.T) context.Context {
```

Comment:
I think we have `testutil.Context()` method, which is similar to this.

Reply:
Fixed